backend/src/models/schemas/chat.py

```diff
@@ -25,8 +58,21 @@ class Session(BaseSchemaModel):
 
 
 class ChatHistory(BaseSchemaModel):
+    """
+    Object for the response body of the chat history endpoint.
+
+    Attributes:
+    -----------
+    sessionUuid: str
+        Session UUID
+    chat_type: str
+        Type of chat
+    message: str
+        Message
+    """
+
     sessionUuid: str = Field(..., title="Session UUID", description="Session UUID") 
-    type: str  = Field(..., title="Type", description="Type") 
+    chat_type: str  = Field(..., title="Chat Type", description="Type of chat") 
```

Comment:
@Micost I replace the type here. type is a keyword in Python. So, we replaced it to chat_type.